### PR TITLE
README.md: add user documentation for Masq-lib

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,109 @@
...
+
+#### `masq.signout() -> Promise<void>`
+
+Signs out of Masq and disconnects from Masq.
+
+#### `masq.logIntoMasq(stayConnected) -> Promise<ret: object>`
+
+`stayConnected` is a boolean representing if the User should stay connected until he explicitely signs out.
+
+The function returns an object containing an attribute `link` containig the Url the user has to access to connect to Masq.
```
Comment (G-Ray) [inline README.md]: containig -> containing

File: README.md [hunk 2]
```diff
@@ -1,11 +1,109 @@
...
+
+Returns a boolean, true if a User is already logged into Masq and if the User-app is already connected to Masq.
+
+#### `masq.connectToMasq() -> Promise<void>`
+
+Connects to Masq if a User is already logged into Masq.
+
+#### `masq.disconnect() -> Promise<void>`
+
+Disconnects from Masq without logging out.
```
Comment (G-Ray) [inline README.md]: Can you add details to what disconnecting means ?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,109 @@
...
+
+Returns a boolean, true if a User is already logged into Masq, false if not.
+
+#### `masq.isConnected() -> connected: boolean`
+
+Returns a boolean, true if a User is already logged into Masq and if the User-app is already connected to Masq.
+
+#### `masq.connectToMasq() -> Promise<void>`
+
+Connects to Masq if a User is already logged into Masq.
```
Comment (G-Ray) [inline README.md]: Can you add just a sentence which describe what 'logged into Masq' mean please ? Like: `(e.g: isLoggedIn() returns true)`

File: README.md (hunk 1 above)
Comment (sujeebant) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (sujeebant) [reply README.md]: Created a Pr to stop exposing the function disconnect as it might not be useful to expose it in its current form

**Pull request merged**